cli/src/browser.rs

```diff
@@ -20,6 +20,7 @@ use browser_utils::{
 	Client,
 	browser_configuration, set_console_error_panic_hook, init_console_log,
 };
+#[cfg(feature = "full-node")]
 use std::str::FromStr;
```

Comment:
```suggestion
```

This makes no sense... The browser node will not have the `full-node` feature activated.

Reply:
This comes from https://github.com/paritytech/polkadot/pull/1795/commits/f3fd10fa9016e8525f2ffc6761de3cfbe3050f86; @montekki showed that it reduced warnings when building the browser target.